_toc.yml

```diff
@@ -138,9 +138,9 @@ entries:
             - file: docs/tools/terraform/reference/cookbook/postgresql-custom-configs-recipe
               title: PostgreSQL with custom configs
             - file: docs/tools/terraform/reference/cookbook/kafka-mongodb-recipe
-              title: Apache Kafka MongoDB Source Connector  
-            - file: docs/tools/terraform/reference/cookbook/kafka-custom-conf-recipe.rst
```

Comment:
Why are we removing the custom configuration one?

Reply:
That recipe and TOC entry weren't there when this PR went in. Re-added back. Thanks.